modules/rostests/winetests/ntdll/threadpool.c

```diff
@@ -20,71 +20,95 @@
 
 #include "ntdll_test.h"
```

Comment:
IMO the winetest import should be either a separate commit or part of the import of the original wine code. It doesn't fit into this commit.